### DISCUSSION TITLE
WizardForm goes back a route when you have a radio input selected and you press enter

### DISCUSSION THREAD

**Discussion by aimeetacchi** [react-hook-form/react-hook-form#9001] [General] [answered]
I was wondering if anyone knew why when you have a WizardForm and you have a Radio input on each route, you select one when you hit enter it takes you back a route. Is there a way to turn this enter pressing off so it doesn't take you back to the previous route with the previous question on? thanks

it doesn't appear to be submitting, which I don't want it to do anyway. I have a button I want the user to click at the last route which all works fine.

**Comment by bluebill1049** [comment]
share a codesandgox?

**Reply by aimeetacchi** [reply]
hello here's a sandbox - https://codesandbox.io/s/my-react-hook-form-wizard-93782g

if you go to like question-2 click a radio then press enter it takes you back to question-one. how can I stop this?

**Comment by leapful** [comment, marked as answer]
@aimeetacchi : HTML element <button> will be in type "submit" by default therefore key press "Enter" will trigger the "Back" button as it's the only button in the form ("Next question" is a an anchor <a>, not a button).

Referrer: https://stackoverflow.com/questions/31644723/what-is-the-default-button-type

Setting type "button" for "Back" button will solve the issue.

Working demo: https://codesandbox.io/s/my-react-hook-form-wizard-forked-0lli5u

**Reply by aimeetacchi** [reply]
oh awesome, Thanks very much for answering my question!!
